### PR TITLE
Replace format() function with f-strings on asv/benchmark.py

### PR DESCRIPTION
**Pull request by LucyJimenez**
xref https://github.com/airspeed-velocity/asv/issues/1077
Change f-strings instead `.format()` on `asv/benchmark.py`

### CODE REVIEW

**Comment by LucyJimenez** [comment]
@datapythonista Do you know the best way to replace the `.format()` function in those cases? Could you please help me?

image

image

**Comment by dorothykiz1** [comment]
@LucyJimenez, I think you can declare with a variable and call that variable in the `f string `to make it more readable

**Comment by datapythonista** [comment]
> @datapythonista Do you know the best way to replace the `.format()` function in those cases? Could you please help me?
> 
> image
> 
> image

Can you add this as a comment in the relevant code, instead of having an image, so I can comment over the code, or copy it, please?

I'd probably use `'    ' + src.replace(...)`

File: asv/benchmark.py [hunk 1]
```diff
@@ -344,7 +342,8 @@ def get_source_code(items):
             src = "class {0}:\n    {1}".format(
                 class_name, src.replace("\n", "\n    "))
         elif class_name:
-            src = "    {0}".format(src.replace("\n", "\n    "))
+            nl = '\n'
+            f"    {src.replace(nl, nl+'    ')}"
```
Comment (datapythonista) [inline asv/benchmark.py:346]: This doesn't make sense.

Comment (datapythonista) [inline asv/benchmark.py:346]:
```suggestion
            src = "    " + src.replace("\n", "\n    ")
```

**Comment by datapythonista** [comment]
Thanks @LucyJimenez

**Pull request merged**